.github/workflows/build.yml

```diff
@@ -5,44 +5,25 @@ on:
     branches: [ master, develop ]
   pull_request:
     branches: [ master, develop ]
-
+  workflow_dispatch:
 jobs:
   build-clients:
-    runs-on: windows-2019
```

Comment:
Think twice before you change the specific version tag to the latest tag.

Reply:
windows-2019 doesn't have e.g. .NET 6 SDK which is required.
The workflows have been tested.

Reply:
I mean, even if you tested the workflow **now**, the same workflow file will lead to a different result in the future, because `latest` is specified, instead of the latest version number.